feat: add landing page and proper SVGs

MagMagnita opened the pull request:
![Scrennshot landing page](https://github.com/user-attachments/assets/5a6ba667-8037-4cb4-92f4-e9104e4e3334)

closes #44

milljoniaer commented on frontend/app/pages/welcome/welcome.tsx:
```diff
@@ -1,75 +1,53 @@
-        </span>
-      </Button>
-    </Layout>
+      <div className="p-4 w-full flex justify-end">
```
personally, I would rather center this link, what do you think?

MagMagnita replied:
Sure we can center it

MagMagnita commented on frontend/public/old.ico:
Yes I made a new one with the black stars on a white round background. Just left the old on in if you dont like the new one

MagMagnita commented on package-lock.json:
Oh yes I guess we can add the package-lock to the git ignore

milljoniaer replied:
Yes, we already have on in the frontend sub-directory, I dont see why we need one in the root directory

milljoniaer commented on frontend/app/components/ui/button.tsx:
```diff
@@ -19,7 +19,10 @@ const buttonVariants = cva(
         ghost:
           "hover:bg-accent hover:text-accent-foreground dark:hover:bg-accent/50",
-        link: "text-primary underline-offset-4 hover:underline",
+        link:
+          "text-primary underline-offset-4 hover:underline",
+        white:
+          "bg-white border-2 border-black shadow-xs hover:bg-secondary/100"
```
I would really like to stick to limiting the number of button versions, because it brings no additional value, but makes it only harder to maintain consistency.

If you really think, that the `outline` variant does not work here, what do you think about using a slightly darker grey for borders in general (see app.css), which will than be for all borders consistent.

e.g. `oklch(0.75 0.013 255.508);`

MagMagnita replied:
Changing the border color is still not working for me, are you fine with just chaning the outline button to border-black?

milljoniaer commented on frontend/app/pages/welcome/welcome.tsx:
```diff
@@ -1,75 +1,53 @@
-        vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
-        gubergren, no sea takimata
-      </p>
+    <div className="flex flex-col min-h-screen">
```
thanks :)

LeonardoZambranoL commented:
The logo looks blurry on a bigger screen. Since it is a png and we are likely not getting an SVG, I would just remove it.

![grafik](https://github.com/user-attachments/assets/9a75f236-4508-46b2-a92a-e821ae301998)

LeonardoZambranoL commented:
The description looks empty. On the figma it looked better as it had more text, but yet again it was just lorem ipsum.

I'd say we make the logos smaller (also helps so they dont look as blurred) and add some features of the app as cards. Centering the contet might also help.
![grafik](https://github.com/user-attachments/assets/9e64deb8-6d2a-47d8-b8dc-ab0e9ba4bc64)

milljoniaer commented on frontend/app/pages/welcome/welcome.tsx:
```diff
@@ -1,75 +1,52 @@
+
+      <div className="p-8 flex flex-col flex-grow">
+        <Text hierachy={1}>Welcome to MediaMind!</Text>
+        <p className="p-4 block w-full text-lg leading-relaxed">
+          Stay ahead in the fast-changing world of media with our Automated
+          Press Analysis Tool. Keep your client teams informed and prepared with
+          daily press reviews, personalized to your needs. Our platform extracts
+          and compiles the most relevant news articles into concise, anonymized
+          reports – delivered directly to your inbox each morning. Save time,
+          stay informed, and focus on what matters most – delivering value to
+          your clients.
+        </p>
```
Yes, exactly 👍

milljoniaer commented:
@MagMagnita so just a quick summary of everything we discussed:

- please remove the root package-lock.json, if we dont need it
- use the outline button (adjust color of borders in app.css if needed)
- place text in the `<Text>` component (add more typography in there if needed)
- use the Layout component for everything (except the hero image)
- add the feature cards as sketched by Leonardo

Thank you! 🚀

MagMagnita commented:
I incorporated all changes.
- The Liner erros were not prettier related. I added an ignores tag to the linter config for the router files.
- I had to slightly change the layout and text component to use it for the landingpage. Also added className as prop for the text for better formatting.
- If you have more questions, write me on slack I am available the whole day Thursday and Friday for faster feedback! :)

MagMagnita commented on frontend/app/components/ui/button.tsx:
```diff
@@ -14,7 +14,7 @@ const buttonVariants = cva(
           "bg-destructive text-white shadow-xs hover:bg-destructive/90 focus-visible:ring-destructive/20 dark:focus-visible:ring-destructive/40 dark:bg-destructive/60",
         outline:
-          "border bg-background shadow-xs hover:bg-accent hover:text-accent-foreground dark:bg-input/30 dark:border-input dark:hover:bg-input/50",
+          "bg-background shadow-xs hover:bg-accent hover:text-accent-foreground dark:bg-input/30 dark:hover:bg-input/50 border-2 border-black",
```
We can also change the border for outlines later on if it looks better. We can also let the others decide, I am fine with both!
